lesson6giang/src/main/java/com/example/lesson6giang/BlankFragment.java

```diff
@@ -0,0 +1,144 @@
+package com.example.lesson6giang;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.*;
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+import static com.example.lesson6giang.AddItemActivity.*;
+
+
+public class BlankFragment extends Fragment {
+    private static final String TAG = BlankFragment.class.getName();
+    public static final Integer CREATE_STUDENT_REQUEST = 1;
+
+    RecyclerView recyclerView;
+    StudentAdapter adapter;
+    ArrayList<Student> students = new ArrayList<Student>();
+
+    private static final Integer CREATE_MODEL_REQUEST = 1;
+
+    FloatingActionButton btnAdd, btnSort;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+       View rootView = inflater.inflate(R.layout.fragment_blank, container, false);
+       recyclerView = rootView.findViewById(R.id.listStudent);
+
+       addListStudent();
+
+       adapter = new StudentAdapter(students, getActivity());
+       LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getActivity());
+       recyclerView.setAdapter(adapter);
+       recyclerView.setLayoutManager(linearLayoutManager);
+
+       insertLines();
+
+       createGidLayout();
+
+
+        //Xoa
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+        // Inflate the layout for this fragment
+
+        creatButtonClick(rootView);
+
+        return rootView;
+    }
+
+    private void creatButtonClick(View rootView) {
```

Comment:
kiểm tra chính tả và cách đặt tên
```suggestion
    private void createStudentButtonOnClick(View rootView) {
```